contracts/test/helpers/Assert.sol

```diff
@@ -0,0 +1,1403 @@
+// Taken from https://github.com/trufflesuite/truffle/blob/ef03be9e35909e5e7c97a9ecb07ccae4872645ca/packages/truffle-core/lib/testing/Assert.sol
```

Comment:
Typically, if we ran the Solidity tests with the normal Truffle test workflow, the Solidity tests would be able to access this library via `import truffle/Assert.sol`. However, since we are using a workaround that involves running the Solidity tests as a part of the JS tests in the Truffle workflow the Solidity tests are unable to access this library via `import truffle/Assert.sol`. Instead, we manually include this library so that it can be imported by the Solidity tests